install/0000_30_machine-api-operator_08_rbac.yaml

```diff
@@ -50,6 +50,16 @@ rules:
     verbs:
       - get
 
+  - apiGroups:
```

Comment:
Please, extend https://github.com/openshift/machine-api-operator/blob/master/config/machine-api-operator-patch.yaml as well.

Reply:
Addressed in https://github.com/openshift/machine-api-operator/pull/295